sdk/monitor/monitor-opentelemetry-exporter/src/utils/common.ts

```diff
@@ -287,3 +287,10 @@ export function serializeAttribute(value: AnyValue): string {
 export function shouldCreateResourceMetric(): boolean {
   return !(process.env[ENV_OPENTELEMETRY_RESOURCE_METRIC_DISABLED]?.toLowerCase() === "true");
 }
+
+export function isSyntheticSource(attributes: Attributes): boolean {
```

Comment:
While the only 2 types defined are "bot" and "test", would it make more sense to just check if there is a non-whitespace string?

That way when something else is added we just automatically support it, without having to update this code to "match" whatever else gets added in the future...

Reply:
Yeah, I think this makes sense. Allows the semantic convention to dictate what users should pass, but I think here we should just care that the value has been populated at all so we won't need to update this logic in the future.